git-reqs

```diff
@@ -48,6 +48,10 @@ def create_report(args):
 
     if args.type == 'relations':
         exporttools.create_report(project, args.module)
+    elif args.type == 'bokeh':
+        exporttools.draw_bokeh(project, args.req)
+    elif args.type == 'test_coverage':
+        exporttools.draw_coverage_diagrams(project)
 
```

Comment:
Should there be some output if another type is used? I.e. "else print("Type " + args.type + " not supported. Try help to see the possible types.")